iocore/net/quic/QUICFrame.cc

```diff
@@ -28,6 +28,7 @@
 #include "QUICStream.h"
 #include "QUICIntUtil.h"
 #include "QUICDebugNames.h"
+#include "QUICPacket.h"
```

Comment:
Do we need this?

Reply:
Yes, since retransmitted frame will call packet->type(). And QUICPacket.h has removed from QUICFrame.h

Reply:
Ah, I see.

```
QUICRetransmissionFrame::QUICRetransmissionFrame(QUICFrameUPtr original_frame, const QUICPacket &original_packet)                                                                                                                                                                                                                                                                                    
 : _packet_type(original_packet.type()) 
```
Maybe we can remove it completely later because it requires only the packet type.